src/DonationForms/AsyncData/Actions/GetAsyncFormDataForListView.php

```diff
@@ -96,4 +98,13 @@ private function isAsyncRevenue(): bool
     {
         return AdminFormListViewOptions::isRevenueColumnAsync();
     }
+
+    /**
+     * @unreleased
+     */
+    private function isFormGoalEnabled(int $formId): bool
+    {
+        return give_is_setting_enabled(Give()->form_meta->get_meta($formId,
+            DonationFormMetaKeys::GOAL_OPTION()->getKeyAsCamelCase(), true));
```

Comment:
Since this meta is v2 backwards compatible, we may also want to optionally check for the v3 flag specifically:

```php
$donationForm->settings->enableDonationGoal
```